test/unit/profiles/profile_test.rb

```diff
@@ -99,6 +99,15 @@
     end
   end
 
+  describe 'skips loading on unsupported platform' do
+    let(:profile_id) { 'windows-only' }
```

Comment:
If you only use it once, my preference would be to just inline this string, and not use `let`.  Not a blocker.